coralproject/talk
# [CORL-1078] Remove Concurrency

## What does this PR do?

Removes concurrency options in favor of recommending horizontal scaling instead.

## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## How do I test this PR?

Start Coral in production (`npm run start`) without specifying a `CONCURRENCY` parameter. Notice how it starts with a single worker instead!
